python/ql/test/library-tests/ApiGraphs/py3/deftest2.py

```diff
@@ -0,0 +1,19 @@
+# Subclasses
+
+from flask.views import View #$ use=moduleImport("flask").getMember("views").getMember("View")
+
+class MyView(View): #$ use=moduleImport("flask").getMember("views").getMember("View").getASubclass()
```

Comment:
This should be `def=`, right? There's no use of a subclass of `flask.views.View`, but there _is_ a definition.

Reply:
I'm not sure that should be a def-node.  
In my head I'm simplifying to what it would look like in a language that only has functions, and in my head that turns it into a use node.

Reply:
A class definition, of a specific subclass, sounds like a def to me :shrug: 

_I came here summoned to looked at another comment, so just randomly looked at this._

Reply:
Yes, but we also don't have def-nodes for assignments to local variables.   
I'm still not seeing the argument for it being a def node (or how that def node would be used).

Reply:
I guess a plausible (if manufactured off the top of my head) use of such a def node is to express things like "in a definition of a subclass of `FancyFramework.Form`, the CSRF token value is set to `None`". (or logging could be turned on/off, etc..)